README.md

```diff
@@ -1446,6 +1447,7 @@ Update Time, five active automations, webhooks.
   * [figma.com](https://www.figma.com) — Online, collaborative design tool for teams; free tier includes unlimited files and viewers with a max of 2 editors and three projects.
   * [framer.com](https://www.framer.com/) - Framer helps you iterate and animate interface ideas for your next app, website, or product—starting with powerful layouts. For anyone validating Framer as a professional prototyping tool: unlimited viewers, up to 2 editors, and up to 3 projects.
   * [freeforcommercialuse.net](https://freeforcommercialuse.net/) — FFCU Worry-free model/property release stock photos
+  * [freejpg](https://www.freejpg.com.ar/) — Hand-curated free stock photos.
```

Comment:
Not so sure about this one, there are no guidance on who took these photos, model release forms or any of the other things that would be required to make these images actually free. So its very questionable if these images are actually free.

Reply:
Further, no real contact details or anything - you really just cant trust the rights to these being free exist.